Using-the-CLI.md

```diff
@@ -2,7 +2,7 @@
 
 Run `sudo npm install -g grunt-cli`.
 
-The `grunt` command-line interface comes with a series of options. Use `grunt -h` from your terminal to show these options.
+The `grunt` command-line interface comes with a series of options (You may only need to use sudo (for OSX, *nix, BSD etc)). Use `grunt -h` from your terminal to show these options.
```

Comment:
This is an obscure place to add a description explaining sudo; instead, it should be on the same line as the code expressed.